server/src/users/mod.rs

```diff
@@ -0,0 +1,11 @@
+use std::path::{Path, PathBuf};
+
+pub mod policy;
+
+const DEFAULT_USER_ID: &str = "default";
+
```

Comment:
I think we want to abstract a user through its specific data structure, even though it could be very simple for now:

```Rust
pub struct User {
    id: String,
}
```
 
then add a `Default` and a `from_string` implementations for it.

All the `users::policy` functions would become `User` methods, allowing us to change and expand the `User` definition in the future without having to modify the API and its callers.